presto-main/src/main/java/io/prestosql/sql/gen/CompilerOperations.java

```diff
@@ -65,6 +65,9 @@ public static boolean in(Object value, Set<?> set)
     public static boolean testMask(@Nullable Block masks, int index)
     {
         if (masks != null) {
+            if (masks.isNull(index)) {
+                return false;
+            }
```

Comment:
```
return !masks.isNull(index) && BOOLEAN.getBoolean(masks, index);
``` 
?

Reply:
I thought the way I wrote it was more readable as it makes the intended behavior and sequencing of operations explicit.